mrp_unbuild_view_improves/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "MRP Unbuild View Improves",
+    "summary": """
+        Adds some improvements for Unbuild views
+    """,
+    "author": "Solvos",
+    "license": "LGPL-3",
+    "version": "13.0.1.0.0",
+    "category": "Manufacturing",
+    "website": "https://github.com/solvosci/slv-manufacture",
+    "depends": ["mrp", "mrp_unbuild_advanced"],
```

Comment:
`mrp_unbuild_advanced` dependency should be avoided. It's the opposite situation: we should consider moving some current `mrp_unbuild_advanced` functionality to this new addon, and make `mrp_unbuild_advanced` dependent of this new one, if it was finally required (and only in that case). Please comment if you don't understand philosophy behind it.

Reply:
Calendar view has unbuild_date field as date_start which is a field defined on mrp_unbuild_advanced.
How can we bypass this dependency?

Reply:
Ooops you're right. I'll think a little bit about it

Reply:
Due to this problem, we've finally chosen moving it to `mrp_unbuild_advanced` as a PR, then created #69